tests/common.py

```diff
@@ -299,14 +307,14 @@ def async_add_executor_job(target, *args):
 
         return orig_async_add_executor_job(target, *args)
 
-    def async_create_task(coroutine, name=None):
+    def async_create_task(coroutine, name=None, eager_start=False):
```

Comment:
This should be true if the core is > 2024.5.0, right?

Reply:
Yeah, updated here: https://github.com/hacs/integration/pull/3673